docs/user/document/category-spec.en.md

```diff
@@ -1,18 +1,17 @@
 # Category - Specification
 
-Document category (`category`) with value type `string` of 1 or more characters
-with `pattern` (regular expression):
+Document category (`category`) with value type `string` of 1 or more characters with `pattern` (regular expression):
 
 ```regexp
 ^[^\\s\\-_\\.](.*[^\\s\\-_\\.])?$
 ```
 
-Document category defines a short canonical name, chosen by the document
-producer, which will inform the end user as to the category of document.
+Document category defines a short canonical name, chosen by the document producer, which will inform the end user as to
+the category of document.
```

Comment:
I would expect these two lines to be wun

Reply:
In this case we run into the limit now set to 120 . Should we increase it further?

Reply:
Normally, I would agree (as VSCode does automatic word wrap for me...). It is the same for the Github web editor... so I wonder whether we should deactivate that rule completely :thinking: 

@mfd2007, @valentindey Thoughts?

Reply:
From the side lines: I would not enforce a line limit per automat but leave that to the human reviewers (where it belongs IMO). In my experience co-authors learn fast to break lines in meaningful places when they see the less noisy diffs. Until linters become clever enough to realize more linguistically adequate rules (full stop, comma, and line length combined) requiring human interaction seems to be the lesser of the problems.

Reply:
I do not have a (very) strong opinion on the topic so I removed the limit altogether and reformatted the markdown files in a more semantically correct way.

Reply:
I'm going to resolve this discussion now and merge the branch. Please feel free to create an issue and/or reach out again if you see need for further adjustments.